lib/types.ts

```
@@ -90,68 +132,7 @@ export enum DomainGetXMLDescFlags {
-    hasSnapshot(): Promise<boolean>;
-    
-    getBlockInfo(device: string): Promise<DomainBlockInfo>;
-    getBlockDevices(): Promise<string[]>;
-    
-    getInterfaceAddresses(source?: DomainInterfaceAddressesSource): Promise<DomainInterfaceInfo[]>;
-}
-
-export interface HypervisorConstructor {
-    new (options: HypervisorOptions): Hypervisor;
-}
+export interface Domain {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)